docs/orchestrating-elastic-stack-applications/elasticsearch/update-strategy.asciidoc

```diff
@@ -21,7 +21,7 @@ spec:
       maxSurge: 3
       maxUnavailable: 1
 ----
-`maxSurge`: Refers to the number of extra Pods that can be temporarily scheduled exceeding the number of Pods  defined in the specification. This setting is useful for controlling the resource usage of the Kubernetes cluster when new Pods need to be spun up to replace existing Pods. For example, when applying a configuration change to a Node Set, ECK creates a clone of the Node Set with the desired configuration and shuts down the old Pods. Without the `MaxSurge` restriction, the cluster would temporarily contain twice as many Pods as usual while this operation is in progress.
+`maxSurge`: Refers to the number of extra Pods that can be temporarily scheduled exceeding the number of Pods defined in the specification. This setting is useful for controlling the resource usage of the Kubernetes cluster when new Pods need to be spun up to replace existing Pods. This setting is only relevant when some existing nodeSets are removed and new nodeSets are added. In this case, `MaxSurge` can prevent all Pods from the new nodeSet to be added, before some Pods of the old nodeSets are removed. Without the `MaxSurge` restriction, the cluster would temporarily contain twice as many Pods as usual while this operation is in progress.
```

Comment:
I would probably change
```
the cluster would temporarily contain twice as many Pods as usual while this operation is in progress.
```

to
```
the cluster may temporarily contain a lot more more Pods than requested in the specification
```

since "twice" here is not accurate.